### PR TITLE
Docs for the 1.4.0 release

### PR DESCRIPTION
**Pull request by irsal**
- Fixed identity doc
- Fixed delegator doc
- Added upgrade doc

### CODE REVIEW

File: _docs/delegators.md [hunk 1]
```diff
@@ -1,23 +1,20 @@
...
-subtitle: Stake as a delegator
+title: Delegator Overview
+subtitle: How to stake to a collator and earn rewards
 author: charles
-tags: [delegate, delegator, stake, collotor]
+tags: [delegator]
 ---
 
-# Introduction
+## Risks
```
Comment (irsal) [inline _docs/delegators.md:8]: @chrisli30 - do we need legal to approve this?

File: _docs/upgrade-collator-node.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+---
```
Comment (irsal) [inline _docs/upgrade-collator-node.md:1]: @atomaka - here's the starting doc to do the node upgrade. Please do a separate PR against this branch to patch.

File: _docs/identity.md [hunk 3]
```diff
@@ -1,73 +1,86 @@
...
+
 After the extrinsic is confirmed, there will be a gray minus sign to the left of the account name, indicating that the account has been registered but has not yet passed the judegement.
 
 ![no-judgements](../../assets/img/identity/no-judgements.png)
 
-## Request Judegement
+### Step 4: Request a judgement
 
-### Retrieving the List of Registrars
+Navigate to **Developer > Extrinsics**, and use your account to send a `identity.requestJudgement` extrinsic.
```
Comment (atomaka) [inline _docs/identity.md:30]: Do we expect to allow more registrars than just our own?  If not, we could be specific in this documentation with our index and fees (if any).

**Review by atomaka — COMMENTED** [review]
I'll open a PR with what I suggested for Docker.

File: _docs/collators.md [hunk 4]
```diff
@@ -139,3 +184,23 @@ For example, in Turing, if you've joined as a collator candidate, and you are co
...
+New session keys are used after the current session completes and two full sessions elapse.
+
+### Under what circumstances would I be removed from the candidate pool?
+Currently slashing has not been implemented on the Turing network. You will only be removed from the candidate pool if you initiate leaving the pool through extrinsic.
+
+### I want to run multiple collators, can I use the same wallet?
+No. Each collator must be associated with a unique wallet in order to join the candidate pool. 
+
+### What rewards will I get?
+Collators collect 2 types of rewards: the collator reward and the staking reward. 1% annual inflation goes directly to collators. This does not dilute based on the number of people staking to your node. 2.5% annual inflation goes to staking rewards. This is split proportionately among all stakers to that node. This will be diluted as more people stake to your node. This creates an incentive for roughly all the nodes to have similar stakes for stakers to maximize their APY.
```
Comment (nvengal) [inline _docs/collators.md:198]: These numbers are based off of the current inflation (5%). If we ever change that these will become inaccurate.

If we want to decouple from the 5% inflation, we could present these numbers differently- 20% of the annual inflation goes to collators, 50% of the annual inflation goes to staking rewards.

My suggestion is easier to maintain in the long term, but if you think this is easier for users to understand in the moment then I'm good with the current text.

File: _docs/collators.md [hunk 5]
```diff
@@ -31,9 +31,43 @@ If you have any questions or run into issues, head over to the [OAK Discord](htt
 | Leave candidacy duration `const parachainStaking.leaveCandidatesDelay`                    | 24 rounds or ~48 hours                                     |
 | Reduction of self-delegation bond duration `const parachainStaking.revokeDelegationDelay` | 24 rounds or ~48 hours                                     |
 | Rewards payout `const parachainStaking.rewardPaymentDelay`                                | Time left to complete current round + 2 rounds or ~4 hours |
+| Inflation `parachainStaking.inflationConfig`                                | 5.00% |
```
Comment (nvengal) [inline _docs/collators.md:34]: Could point out that it's 5% annual, since there are a couple different inflation percents.

File: _docs/collators.md [hunk 6]
```diff
@@ -31,9 +31,43 @@ If you have any questions or run into issues, head over to the [OAK Discord](htt
...
 
 _Note: The source of truth for the values above is the chain state and constants, so please query that to double-check the values_
 
+## Helpful query to get the chain state
+You need to query the chain state a number of times for these operations. With that, we've provided a helpful script for you to use. Navigate to the **Developer > Javascript** tab on the [PolkadotJS App](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.turing.oak.tech#/js).
+
+```javascript
```
Comment (nvengal) [inline _docs/collators.md:41]: Tested on tur staging. Output looks reasonable.
```
------ HELPFUL CHAIN STATE DATA ------
Minimum Collator Bond: 20000000000000000
Number of selected candidates: 5
Round Length: {"current":170,"first":101400,"length":600}
Leave candidacy duration: 24
Reduction of self-delegation bond duration: 24
Rewards payout: 2
Inflation: {"expect":{"min":0,"ideal":0,"max":0},"annual":{"min":50000000,"ideal":50000000,"max":50000000},"round":{"min":11132,"ideal":11132,"max":11132}}
------ HELPFUL COLLATOR ACTIONS DATA ------
Total Candidates in pool: 2
Your Delegation Count: 6
```

File: _docs/collators.md [hunk 7]
```diff
@@ -96,6 +130,10 @@ Similar to Step 3, you'll have to navigate to the chain state page, and call `pa
 
 Top delegated collators will be part of the active set, which is set / chosen at the beginning of each round.
 
+### Step 6 (Optional, but recommended): Setup your on-chain identity
+
+To increase your chances of community members [staking](../delegator) to your collator, we recommend you setup an [on-chain identity](../identity).
```
Comment (nvengal) [inline _docs/collators.md:135]: Staking link is broken.
```suggestion
To increase your chances of community members [staking](../delegators) to your collator, we recommend you setup an [on-chain identity](../identity).
```

File: _docs/delegators.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+## How to stake
+
+### Step 1: Figure out who you want to stake to
+
+Before staking tokens to collators, we recommend you do your research on who to stake towards. To figure out the candidate pool, you can find the set of collators in [Subscan](https://turing.subscan.io/validator). Save their wallet address as `COLLATOR_WALLET_ADDRESS` to be used on the step below.
+
+### Step 2: Figure out your inputs
+To be stake your delegator via PolakdotJS extrinsics, you'll need to figure out three numbers. We've provided a helpful script for you to use. Navigate to the **Developer > Javascript** tab on the [PolkadotJS App](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.turing.oak.tech#/js).
+
+```javascript
```
Comment (nvengal) [inline _docs/delegators.md:24]: Also works.
```
2a. Minimum amount to be staked: 500000000000
2b. Candidate delegation count: 0
2c. Delegation count: 0
```

Comment (nvengal) [reply _docs/delegators.md:24]: Since both collators and delegators are seeing naked tur numbers with all zeroes attached we should probably have an explainer somewhere for newbies and disclaimer about the number of 0s in tur for everyone. (Doesn't need to be included here, just something I'm assuming will come up)

File: _docs/delegators.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+---
+## How to stop staking to a collator
+
+### Step 1: Schedule Request to Stop Delegations
+
+To delegate a candidate, send `parachainStaking.scheduleRevokeDelegation` extrinsic by the following information:
+
+![schedule-revoke-delegation](../../assets/img/staking-delegation/schedule-revoke-delegation.png)
+
+Once you have scheduled an exit delay(24 rounds * 600 block, 12 seconds per block in current Turing Network), you must wait an exit delay before you can then execute it.
```
Comment (nvengal) [inline _docs/delegators.md:57]: Probably worth being explicit that funds don't actually unlock until you perform the second extrinsic to execute the exit.

File: _docs/identity.md (hunk 3 above)
Comment (nvengal) [reply _docs/identity.md:30]: Do you need to request judgement to get a judgement? We're pointing them to discord in the next step.

File: _docs/identity.md [hunk 10]
```diff
@@ -1,73 +1,86 @@
...
+![request-judgement](../../assets/img/identity/request-judgement.png)
 
-As shown above, there are currently two registrars.
+- regIndex: The index of registrars.
+- maxFee: You need to provide a fee in `Planck` higher than registrar.
+
+### Step 5: Contact a registrar to acquire the judgement
 
-The fee of the first registrar (6Ah9LrGFyc4AT22dYUQTuSY8vxJkegZ5F5Udqx4eVJh8AJ6T) is 0, which means that he charges 0 tokens to do a judgement.
+You can either contact the registrar via their on-chain information or you can ask the registrar to review your request in [OAK Discord room](https://discord.gg/7W9UDvsbwh).
```
Comment (nvengal) [inline _docs/identity.md:39]: It would be good to specify a discord channel for folks to send their registrar requests to so that we can monitor it if that's a concern.

File: _docs/identity.md [hunk 11]
```diff
@@ -1,73 +1,86 @@
...
+### Which registrar should I ask for a judgement?
 
-- Low Quality: The information is low quality or imprecise, but can be fixed with an update.
+You will have to select which registrar to pick by vetting their identity. To do so, do some research on their account. You can start by navigating to [Turing's Subscan Accounts](https://turing.subscan.io/account) page and searching for their account.
 
-- Erroneous: The information is erroneous and may indicate malicious intent.
+![registrars](../../assets/img/identity/registrars.png)
 
-## Check your indentity
+As shown above, there are currently two registrars.
```
Comment (nvengal) [inline _docs/identity.md:82]: These screenshots and info are from turing staging. Might be confusing when folks are expecting turing proper. Especially since there are specific wallet addresses in this guide.

Not a high priority to fix, but maybe something to consider updating in the future.

File: _docs/delegators.md (hunk 1 above)
Comment (chrisli30) [reply _docs/delegators.md:8]: We can ask them to take a look once we publish this page. I think it’s not required since there’s no "sale" stuff in it.

File: _docs/identity.md (hunk 3 above)
Comment (irsal) [reply _docs/identity.md:30]: good call, edited

File: _docs/collators.md (hunk 4 above)
Comment (irsal) [reply _docs/collators.md:198]: I'll do a separate PR for this

File: _docs/delegators.md (hunk 9 above)
Comment (irsal) [reply _docs/delegators.md:57]: I'm gonna remove this for now. I wanna test first

**Review by atomaka — APPROVED** [review]
